### DISCUSSION TITLE
Discord server for communication (dev and users)

### DISCUSSION THREAD

**Discussion by Enubia** [cpvalente/ontime#533] [Discussion]
Is there a communication tool present, like Discord, for devs and users?
If not, is it planned?

From a dev POV, it would make things easier like getting info about where certain things are implemented. How certain things should look and so on.

**Comment by cpvalente** [comment]
Hi @Enubia, thank you for this.

In short, there is no real Ontime presence anywhere, this is unlikely to change. The app has achieved a good user base despite my poor (non-existing) efforts in advertisement.

For people reading this in future, hoping for some guidance:
My email is on the project website and my github profile, both devs and users are welcome to reach out anytime. In fact, I maintain good contact with several users who advice on feature development.
I also attempt to maintain some sort of presence in the video engineering discord channel and subreddit

For developers we maintain contact often through slack and discord. You are likely able to find someone for help with Ontime in the video engineering discord channel
If you would like to discuss features, please reach out on mail@getontime.no and we can find a convenient method to communicate.

**Comment by cpvalente** [comment]
For anyone looking into this in future, we have created a discord channel, the links are available in app and documentation
